src/displayapp/screens/Tab.h

```diff
@@ -1,22 +1,24 @@
 #pragma once
 
-#include <cstdint>
 #include "Screen.h"
-#include <bits/unique_ptr.h>
 #include <lvgl/src/lv_core/lv_style.h>
+#include <lvgl/src/lv_core/lv_obj.h>
 
 namespace Pinetime {
+  namespace Components {
+    class Gfx;
+  }
   namespace Applications {
     namespace Screens {
       class Tab : public Screen {
         public:
           explicit Tab(DisplayApp* app, Components::Gfx& gfx);
           ~Tab() override;
-          void Refresh(bool fullRefresh) override;
+          void Refresh(bool fullRefresh);
           void OnObjectEvent(lv_obj_t* obj, lv_event_t event);
 
-        private:
-
+        protected:
+          Components::Gfx& gfx;
```

Comment:
This isn't very nice compared to the rest and when Components will contain more members it's going to have to be moved back

Reply:
Yeah there I had some question marks regarding the whole thing. The Tab class isn't used anywhere yet, I wanted to keep it compile-able though, so I added the Gfx object.  As it looks like a base class for me, I set it as protected, so that it can be used by child-classes.

Reply:
I confirm : the Tab class is not used/built anymore and I can be safely removed. The class Gfx is not used in the main firmware either. I used this class to draw pixels on the display using the class before I integrated LVGL. 
But do not remove Gfx, it is still use as a lightweight UI class in pinetime-graphics

So, if you added this protected variable only for the tab class, I think you can safely remove the variable **and** the Tab Screen :)